tests/test_api_endpoints.py

```diff
@@ -38,7 +38,7 @@ def test_status():
         assert response.json["status"] == "Application is running!"
 
 
-def test_is_configured(auth_token, db_setup, db_create_account, db_create_system):
+def test_is_configured(auth_token, db_setup, db_create_account, db_create_system, db_create_performance_profile):
     with app.test_client() as client:
         response = client.get('/api/ros/v1/is_configured', headers={"x-rh-identity": auth_token})
         assert response.status_code == 200
```

Comment:
@apuntamb Could you add an assert comparison in the `test_is_configured`, between count from systems endpoint and is_configured?
Also I think we can move `test_systems` above `test_is_configured`, since we will need systems test to run fine before pytest moves on to `test_is_configured`